bin/cli.js

```diff
@@ -49,7 +49,7 @@ const projectCommands = require('../commands/project');
 const themeCommand = require('../commands/theme');
 const moduleCommand = require('../commands/module');
 const configCommand = require('../commands/config');
-const accountsCommand = require('../commands/accounts');
+const accountCommand = require('../commands/account');
```

Comment:
nit, but can these be `accountCommands` and `secretCommands`? Helps identify them as command buckets